quarkus-modules/quarkus-hibernate-reactive/src/test/java/com/baeldung/web/DepositResourceUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package com.baeldung.web;
+
+import com.baeldung.entity.Deposit;
+import com.baeldung.repository.DepositRepository;
+import io.quarkus.test.junit.QuarkusTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@QuarkusTest
+public class DepositResourceUnitTest {
```

Comment:
```suggestion
public class DepositResourceIntegrationTest {
```

This is more of an integration test than a unit test since we are doing over-the-wire testing